### DISCUSSION TITLE
change the default click behaviour for music tracks

### DISCUSSION THREAD

**Discussion by vk2him** [music-assistant/support#404] [Feature requests and ideas]
The default action when left clicking a song title is to immediately play the track. This has caught me out a few times on my Macbook as a single one finger tap is a left click and a double finger tap is a right click.  I wanted to look at the detail on the song but inadvertently played it.

  My preference would be to have a user selected option to reverse the left/right click action.

**Comment by OzGav** [comment]
Old v1 question
